strings/camel_case_to_snake_case.py

```diff
@@ -0,0 +1,80 @@
+"""
+this function focuses on converting camel case (eg: camelCase)
+to snake case (snake_case)
+"""
+
+import sys
+
+
+def to_snake_case(word: str) -> str:
+    """
+    This to_snake_case function removes all the vowels
+    from a given sentence or a word
```

Comment:
That's not what the function does though, is it?

Reply:
Thanks for your comment. I have made the necessary changes.